api/src/pcapi/routes/pro/educational_institution.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+import logging
+
+from flask_login import login_required
+
+from pcapi.core.educational import repository as educational_repository
+from pcapi.models.api_errors import ApiErrors
+from pcapi.routes.apis import private_api
+from pcapi.routes.pro import blueprint
+from pcapi.routes.serialization import educational_institutions
+from pcapi.serialization.decorator import spectree_serialize
+
+
+logger = logging.getLogger(__name__)
+
+
+@private_api.route("/institution/<uai>", methods=["GET"])
+@login_required
+@spectree_serialize(
+    response_model=educational_institutions.EducationalInstitutionsWithBudjetResponseModel,
+    on_success_status=200,
+    on_error_statuses=[401],
+    api=blueprint.pro_private_schema,
+)
+def get_educational_institutions_with_budget(
+    uai: str,
+) -> educational_institutions.EducationalInstitutionsWithBudjetResponseModel:
+    institution = educational_repository.find_educational_institution_by_uai_code(uai)
+    if institution is None:
+        raise ApiErrors({"institution": ["Aucune institution trouvée pour cet uai."]}, status_code=404)
+
+    deposits = institution.deposits
+    for deposit in deposits:
```

Comment:
(détail) La variable `deposits` n'apporte pas grand chose, on pourrait faire sans : 

```suggestion
    for deposit in institution.deposits:
```